source/uwp/AdaptiveCards.sln

```diff
@@ -1,7 +1,7 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
```

Comment:
Do not check in the changes to this file.

Reply:
Yes,

---
In reply to: [151219045](https://github.com/Microsoft/AdaptiveCards/pull/871#discussion_r151219045) [](ancestors = 151219045)